definitions/subannual/representative_time_slices.yaml

```
@@ -0,0 +1,38 @@
+# description of represnetative periods
+
+- Average week:
+    duration: 168 hours
+    description: average week over a yearly period for the given variable ie. average value of the 52 weekly values of the variable
+
+- Average week|{Season Input}:
+    duration: 168 hours
+    description: average value of the weekly values of the variable, for the weeks belonging to <season>, which are defined as the weeks with at least 4 days belonging to the considered season
+    
+- hhh|Average week:
+    duration: 1 hour
+    description: hhh is the index of each of the ith hour of the average week. An average week will be fully described by 168 rows with the value of the variable at each hour of the week
+    note: a week starts on Monday and ends on Sunday
+    
+- hhh|Average week|{Season Input}:
+    duration: 1 hour
+    description: hhh is the index of each of the ith hour of the average week. An average week will be fully described by 168 rows with the value of the variable at each hour of the week
+    note: a week starts on Monday and ends on Sunday
+    
+- Representative week:
+    duration: 168 hours
+    description: Representative week over a yearly period for the given variable ie. value of one of the 52 weekly values of the variable, chosen by the data owner
+    note: Representative Week|<criteria> can be defined, eg. with criteria=holiday
+
+- Representative week|{Season Input}:
+    duration: 168 hours
+    description: Representative week over a yearly period, chosen amongst weeks belnging to <season> for the given variable 
```

Use `{}` so that the description from the tag-list ist automatically filled in.

```suggestion
    description: Representative week over a yearly period, chosen amongst weeks belnging to {season} for the given variable 
```